src/components/OSCALCatalogGroup.js

```diff
@@ -30,20 +32,58 @@ const StyledControlDescriptionWrapper = styled("div")`
 
 function CollapseableListItem(props) {
   const [open, setOpen] = React.useState(false);
+  const [listItemOpened, setListItemOpened] = React.useState(false);
 
   const handleClick = () => {
     setOpen(!open);
   };
 
+  const handleFragment = useCallback(() => {
+    if (!listItemOpened) {
+      const { hash } = window.location;
```

Comment:
Can we use the `useLocation` hook?

Reply:
I like this suggestion as it feels like the more "React approach" to the problem. This will remove all linter errors, however, React Testing Library is not very happy. I'm not entirely sure if this is a better choice to go at this point (over `window.location`) as I haven't dug too deep into the test failures, but it seems like the main issue is that `useLocation()` must be used in the context of a route. I've committed my changes as this is one of the more major changes I've made but can revert it if need be.

Reply:
I ended up just refactoring a portion of the code in order to acquire the fragment in the context of a route (in App) and just passed down the fragment's state to `OSCALLoader` for other components to handle. This seems to resolve all cases of: `[Error: useLocation() may be used only in the context of a <Router> component`, while also avoiding linter warnings that were previously added in this PR.